zebra-chain/src/orchard.rs

```diff
@@ -22,4 +23,13 @@ pub use address::Address;
 pub use commitment::{CommitmentRandomness, NoteCommitment, ValueCommitment};
 pub use keys::Diversifier;
 pub use note::{EncryptedNote, Note, Nullifier, WrappedNoteKey};
+pub use orchard_flavor_ext::{OrchardFlavorExt, OrchardVanilla};
 pub use shielded_data::{AuthorizedAction, Flags, ShieldedData};
+
+pub(crate) use shielded_data::ActionCommon;
+
+#[cfg(feature = "tx-v6")]
```

Comment:
I like `tx-v6` better because I think we just need the feature to ensure that v6 transactions can't be (de)serialized on versions that aren't ready to deploy NU7, but either seems fine.